docs/src/pages/themer/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Themer from "@kiwicom/orbit-themer";
+
+import DocLayout from "../../components/DocLayout";
+
+export default ({ location }) => {
+  return (
+    <DocLayout
```

Comment:
Instead of `DocLayout`, I suggest rendering it inside a different Layout, without SideNavigation. It will visually look better in my opinion, links in the navbar and logo should be enough to navigate, plus breadcrumbs are there.

Reply:
Done. Although the navbar links are not there, because they're set to only be displayed on the homepage. Should we change that rule...?

Reply:
yeah, i think for themer it makes sense :)